msteams-platform/concepts/device-capabilities/device-capabilities-overview.md

```diff
@@ -1,19 +1,20 @@
 ---
 title: Overview of device capabilities
 description:  Overview of native device  capabilities.
-keywords: camera image media microphone capabilities native device permissions 
+keywords: camera image media microphone qrcode barcode scanner capabilities native device permissions 
 ms.topic: overview
 ---
 
 # Device capabilities 
 
-Microsoft Teams platform is continuously enhancing developer capabilities aligning with built-in first-party experiences. The enhanced Teams platform allows partners to integrate device capabilities, such as camera, photo gallery, microphone, and location, with their web apps. This integration reduces the barrier to app development, speeds-up development-cycle, and creates new scenarios or use-cases for the developer community.
+Microsoft Teams platform is continuously enhancing developer capabilities aligning with built-in first-party experiences. The enhanced Teams platform allows partners to integrate device capabilities, such as camera, QR or barcode scanner, photo gallery, microphone, and location, with their web apps. This integration reduces the barrier to app development, speeds-up development-cycle, and creates new scenarios or use-cases for the developer community.
```

Comment:
Remove the comma after location.

.... microphone, and location with their web apps.